action.yml

```diff
@@ -25,28 +29,30 @@ inputs:
 runs:
   using: "composite"
   steps:
-    # Setup Python environment
-    - uses: actions/setup-python@v4
+    # Install conda-lock library with Micromamba
+    - uses: mamba-org/setup-micromamba@v1
       with:
-        python-version: '3.11'
-
-    # Install conda-lock library and show version
-    - name: Install conda-lock
-      run: python -m pip install conda-lock
-      shell: bash -l {0}
+        environment-name: conda-lock-env
+        create-args: >-
+          python=3.11
+          conda-lock
+          mamba
 
     # Print debugging information
     - name: Print debug info
       run: |
         conda-lock --version
+        mamba --version
         echo Locking ${{ inputs.file }} file for ${{ inputs.platform }} platform.
       shell: bash -l {0}
 
     # Run conda-lock
     - name: Run conda-lock
       run: |
         rm --force conda-lock.yml
-        conda-lock lock --file ${{ inputs.file }} --kind ${{ inputs.kind }} --platform ${{ inputs.platform }}
+        conda-lock lock $MAMBA --file ${{ inputs.file }} --kind ${{ inputs.kind }} --platform ${{ inputs.platform }}
+      env:
+        MAMBA: fromJSON('{"false": "--no-mamba", "true": "--mamba"}')[inputs.mamba]
```

Comment:
For reference: this conditional environment variable was adapted from https://github.com/orgs/community/discussions/25725#discussioncomment-3248927.